.github/workflows/test.yml

```diff
@@ -15,6 +15,7 @@ jobs:
       - name: Install dependencies
         run: npm ci
       - run: npm run build
+      - run: npx jest -u
```

Comment:
Sorry I explained it wrong, can you run this update of the snapshots on your machine and check in the update so that the unit test pass again?

Reply:
Updated.